README.md

```diff
@@ -541,26 +541,65 @@ However, in some cases it may be useful to use an existing browser profile, for
 example if the webpage you are benchmarking requires being signed into an
 account.
 
-In Chrome, you can use the `user-data-dir` flag to launch the browser using an
-existing profile directory. You may also need to remove the `use-mock-keychain`
-default argument if you encounter authentication problems. You can find out the
-current binary path, profile location, and arguments of a running Chrome session
-by visiting the `chrome://version` URL.
+In Chrome and Firefox, use the `profile` JSON config option to specify an
+existing profile to use. Other browsers do not yet support this option.
 
-NOTE: If there is an existing Chrome process using the profile, you must
-first terminate it. You also need to close all open tabs, or disable the
-"Continue where you left off" startup setting, because tachometer does not
-expect to find any existing tabs.
+#### Chrome
+
+To find your current profile location in Chrome, visit `chrome://version` and
+look for "Profile Path".
+
+If there is an existing Chrome process using this profile, you must first
+terminate it. You also need to close all open tabs, or sable the "Continue where
```

Comment:
Nit: `sable`?

Reply:
Oops, should be "disable". Fixed, thanks!